infra.bs

```diff
@@ -313,6 +313,14 @@ contains, in the range 0x61 (a) to 0x7A (z), inclusive, by 0x20.
 <a>byte sequence</a> <var>B</var>, if the <a>byte-lowercase</a> of <var>A</var> is the
 <a>byte-lowercase</a> of <var>B</var>.
 
+<p>To <dfn export for="byte sequence">map</dfn> a <a>byte sequence</a> <var>input</var> to a
+<a>string</a>, return a string whose <a for=string>length</a> is equal to <var>input</var>'s
+<a for="byte sequence">length</a> and whose code points have the same values as <var>input</var>'s
+bytes, in the same order.
+
+<p class=note>This matches the behavior of a ISO-8859-1 decoder, except no such decoder is defined
+as the Encoding Standard maps "iso-8859-1" to <a>windows-1252</a>. [[ENCODING]]
```

Comment:
We should be pretty cautious with this note as some people I know are very confused/mad by the mapping of iso-8859-1 to windows-1252. I'd rather phrase this in a way that makes it sound less like the Encoding Standard is being "wrong". And I think it's a valuable opportunity to spell out this confusing space for people. Including myself.

Something like

> This process corresponds somewhat to what has historically been called ISO-8859-1 decoding, at least for some ranges of bytes. However, in reality, software that performs such decoding has behavior for bytes in the range 128–256 that goes beyond the ISO-8859-1 specification, leading to the modern practice of aliasing "iso-8859-1" to "windows-1252" and performing the corresponding algorithm from the Encoding Standard. The algorithm here is different, treating all bytes directly as their corresponding code points instead of using the windows-1252 [index](https://encoding.spec.whatwg.org/#index) for bytes in the range 128–256. [[ENCODING]] [[ISO8859]].

([[ISO8859]] would go to https://www.iso.org/standard/28245.html I guess? Which doesn't let you view the PDF without paying? Wikipedia also gives ftp://std.dkuug.dk/JTC1/sc2/wg3/docs/n411.pdf as a link.)

Then we'll need a similar reverse note.